custom/enikshay/ucr/data_sources/episode_v4.json

```diff
@@ -19,19 +19,14 @@
         "table_id": "episode_v4",
         "display_name": "Episode v4",
         "configured_filter": {
-            "type": "and",
-            "filters": [
```

Comment:
will changing the filter still allow this to be updated in place? Otherwise I don't think you should include this change.

Reply:
i dont think it will trigger a rebuild, and if it does it should definitely be in place. but this data source is only used in testing anyway, so im not sure its a problem if we have to rebuild in off hours

Reply:
Rebuilds only happen automatically whenever a column is added or removed or the type changes. Filters have no bearing on rebuilds

Reply:
Wouldn't filter changes change which documents should be included in the datasource (potentially removing some that were there before, and adding others that had already been looked over)?

Reply:
The way we decide to rebuild is to inspect the table that's currently live in the DB and the table that would be created from the UCR config. If those are the same, then we don't rebuild. 

filters are lost in that, but there's actually no way to tell what the old filter was anyways (and thus no way to compare them), because we don't keep old docs around

Reply:
Another thing that gets lost is if you change a calculation of a column. As long as the data type and column_id stay the same you won't get an automatic rebuild. For example if you change it from a case property lookup to a complex expresion you need to manually kick off the rebuild

Reply:
Got it. Didn't know that before. Thanks.